code/modules/mob/living/carbon/human/_species.dm

```diff
@@ -1463,6 +1463,10 @@ GLOBAL_LIST_EMPTY(features_by_species)
 		// Apply the damage to all body parts
 		humi.apply_damage(burn_damage, BURN, spread_damage = TRUE)
 
+	// No cold damage if the body is cryogenically frozen
+	if(humi.coretemperature <= BODYTEMP_CRYO_THRESHOLD)
+		return
```

Comment:
This doesn't really work because it would allow a person who's still alive to not take any damage if they're below a specific temperature.